data/org.mupen64plus.Mupen64Plus.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.mupen64plus.Mupen64Plus</id>
+  <launchable type="desktop-id">org.mupen64plus.Mupen64Plus</launchable>
+  <name>Mupen64Plus</name>
+  <summary>Nintendo 64 Emulator</summary>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <description>
+    <p>
+      Mupen64plus is a Nintendo 64 Emulator.
+      This package includes all the plug-ins.
+    </p>
+  </description>
+  <url type="homepage">http://mupen64plus.org/</url>
+  <url type="help">http://mupen64plus.org/docs/</url>
+  <url type="bugtracker">https://github.com/mupen64plus/mupen64plus-user-issues/issues</url>
+  <url type="donation">https://www.paypal.com/webapps/shoppingcart?flowlogging_id=59a5b20d47d74&mfid=1541895691620_59a5b20d47d74</url>
```

Comment:
Where did you find this PayPal link? I don’t see it on the homepage.

Reply:
It's linked from here (looks like an old site):

http://mupen64.emulation64.com/

Should it be removed from this file?

Reply:
Yes; that PayPal link is for Hacktarux who as far as I know hasn’t been involved for years.